src/slack_api_util.ts

```diff
@@ -288,6 +288,20 @@ export async function updateSlackChannelNamesAndIdsInRedis(
       slackChannelNamesAndIds
     );
   }
+
+  return slackChannelNamesAndIds;
+}
+
+// Get Slack Channel Name / ID map, using cache if needed
```

Comment:
should this be `populating cache if needed`? just wanna confirm my understanding